### PR TITLE
Model test

### PR DESCRIPTION
**Pull request by PichapopRo**
## What I have done.
- Create a test and separate file for each model class test.

### CODE REVIEW

File: webpage/tests/test_recipe.py
Comment (knilios) [inline webpage/tests/test_recipe.py:1]: The test test_get_steps should have tested with more steps. I suggest creating more steps and then testing their order.

**Review by knilios — CHANGES_REQUESTED** [review]
I only have one issue. If this is fixed, it will be ready to merge.

**Comment by PichapopRo** [comment]
I added more step feel free to review

**Review by knilios — APPROVED** [review]
OK

**Pull request merged**